SwiftCheckTests/NSDataSpec.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  NSDataSpec.swift
+//  SwiftCheck
+//
+//  Created by Projector on 12/28/15.
+//  Copyright © 2015 Robert Widmann. All rights reserved.
+//
+
+import SwiftCheck
+
+extension NSData {
+    static var generator: Gen<NSData> {
+        return Gen<NSData>.pure(
+            String.arbitrary.generate.dataUsingEncoding(NSUTF8StringEncoding)!
```

Comment:
Just a note here, this won't do what you think it will do.  You want random data that is different across test cases, so you don't need to call `generate`, just

    return String.arbitrary.fmap { $0.dataUsingEncoding(NSUTF8StringEncoding)! }